### ISSUE TITLE
Add a CSV library to read data tables

### ISSUE THREAD

**Issue by mnislam01** [oservo/aiFi#3]
This will come very handy: https://github.com/winobes/libcsv

Writing data frame in two dimensional arrays for test is simply too much. This will make life easier.

**Comment by eknoyon** [comment]
Obviously. Just prioritized immediately after Optimization Functions. Couple of days away,  these are.
